public/style/components/_animation.scss

```diff
@@ -93,3 +108,22 @@ svg .bolt {
     opacity: 0;
   }
 }
+
+@keyframes sleetMove {
```

Comment:
This is interesting starting at 100%, I didn't know you could do that. Another possibility is just to reverse animation. `animation-direction: normal, reverse;` and start at 0%?